### ISSUE TITLE
[BUG] Error when read PDF file

### ISSUE THREAD

**Issue by DI-hungdh** [maxpmaxp/pdfreader#135]
**Describe the bug**
Get error when trying to process PDF file

**To Reproduce**
Code snippet and steps to reproduce the behavior:
```
fd = open(file_name, "rb")
doc = PDFDocument(fd)
```

Error:
```
unsupported operand type(s) for +: 'NoneType' and 'bytes'
```

**PDF file**
My pdf file is large so i attach link: https://drive.google.com/file/d/17RFmFQD70BHh9W6FKPD3Ze8LfrF0t_gW/view?usp=sharing

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Pdfreader Version: 0.1.15
 - OS: ubuntu-20.04
 - Python: 3.12
 
**Additional context**

**Comment by maxpmaxp** [comment]
A bug specific to linearized PDFs. Thanks for attaching the example. Going to fix soon.

**Comment by maxpmaxp** [comment]
Fixed on #136 - the changes are on master and gonna be in the next release.
